### PR TITLE
Scooter tile styling

### PR DESCRIPTION
**Pull request by wfeiring**
Fixes the buggy scooter tile css and makes it look and behave like the other tiles. There´s a minor pixel between the top of the logo of the scooter operator and the label, but I was not able to get them perfect. Its pretty close though!

### CODE REVIEW

File: src/dashboards/Compact/styles.scss [hunk 1]
```diff
@@ -14,6 +14,14 @@
             overflow-x: hidden;
             margin-bottom: 1rem;
         }
+
+        .scootertile {
+            width: 100%;
+            padding-top: 2rem;
+            overflow-y: hidden;
+            overflow-x: hidden;
```
Comment (draperunner) [inline src/dashboards/Compact/styles.scss]: Du kan slå desse sammen til `overflow: hidden`

File: src/dashboards/Compact/ScooterTile/styles.scss [hunk 2]
```diff
@@ -18,12 +18,14 @@
...
             color: var(--tavla-font-color);
+            font-size: 32px;
         }
     }
 
     &__header-icons {
         display: flex;
         align-items: center;
         font-size: 2rem;
+        color: #babbcf;
```
Comment (draperunner) [inline src/dashboards/Compact/ScooterTile/styles.scss]: Bruk helst css-variabel frå design-systemet.

Comment (draperunner) [reply src/dashboards/Compact/ScooterTile/styles.scss]: Det er sånn kanskje?  `var(--colors-blues-blue60)`

**Review by draperunner — APPROVED** [review]

File: src/dashboards/Compact/styles.scss (hunk 1 above)
Comment (wfeiring) [reply src/dashboards/Compact/styles.scss]: De to overflow-ene eller alle fire? 🤯

Comment (draperunner) [reply src/dashboards/Compact/styles.scss]: overflowene :)

**Pull request merged**